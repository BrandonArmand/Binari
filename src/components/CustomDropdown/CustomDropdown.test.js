import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import CustomDropdown from "./CustomDropdown";

describe("CustomDropdown", () => {
    it("renders snapshot", () => {
        const { container } = render(<CustomDropdown />);
        expect(container).toMatchSnapshot();
    });
    it("renders CustomDropdown", () => {
        render(
            <CustomDropdown buttonText="dropdown">
                CustomDropdown
            </CustomDropdown>
        );
        expect(screen.getByText("dropdown")).toBeInTheDocument();
    });
});
