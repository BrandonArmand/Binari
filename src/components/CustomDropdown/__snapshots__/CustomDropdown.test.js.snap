// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomDropdown renders snapshot 1`] = `
<div>
  <div>
    <div>
      <button
        aria-haspopup="true"
        aria-label="Notifications"
        class="MuiButtonBase-root MuiButton-root makeStyles-button-21 MuiButton-text"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          <b
            class="makeStyles-caret-16"
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
